### ISSUE TITLE
AttributeError: 'Attribute' object has no attribute 'name'

### ISSUE THREAD

**Issue by marmarek** [PyCQA/pylint#2384]
### Steps to reproduce
1. call pylint --rcfile=... modulename

### Current behavior

```
(...)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/pylint/utils.py", line 1159, in walk
    cb(astroid)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/pylint/checkers/base.py", line 1129, in visit_call
    self._check_misplaced_format_function(node)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/pylint/checkers/base.py", line 1120, in _check_misplaced_format_function
    if isinstance(call_expr, astroid.Call) and call_expr.func.name == 'print':
AttributeError: 'Attribute' object has no attribute 'name'
```
See https://travis-ci.com/QubesOS/qubes-manager/builds/80969777#L472-L503

### Expected behavior

No crash.

### pylint --version output

```
pylint 2.1.0
astroid 2.0.2
Python 3.5.5 (...)
```

**Comment by brycepg** [comment]
Thanks, can repro with the following line:

```self.tr().format()```

**Comment by sushobhit27** [comment]
@brycepg ya sure, I will try to fix it soon.

**Comment by PCManticore** [comment]
Closed by #2386
